package.json

```diff
@@ -16,7 +16,7 @@
     "@commitlint/cli": "^15.0.0",
     "@commitlint/config-conventional": "^15.0.0",
     "@monorepo-utils/workspaces-to-typescript-project-references": "2.7.6",
-    "@octokit/core": "3.6.0",
+    "@octokit/core": "4.0.4",
```

Comment:
it started complaining about this dep which was our issue in publishing last week and this was the suggested fix. :shrug: